### PR TITLE
FIX: Add local install of pyart to doc env file

### PR DESCRIPTION
**Pull request by mgrover1**
Add a local install of pyart to docs environment

- [x] Documentation reflects changes

### CODE REVIEW

File: doc/environment.yml [hunk 1]
```diff
@@ -41,3 +41,4 @@ dependencies:
       - pooch
       - versioneer
       - ablog
```
Comment (zssherman) [inline doc/environment.yml:43]: should be:
- -e ..

Comment (zssherman) [reply doc/environment.yml:43]: format was weird but double dashes with a space in between

Comment (mgrover1) [reply doc/environment.yml:43]: fixed

**Comment by mgrover1** [comment]
@zssherman - looks like tests are passing, I am going to merge to get the CI building properly

**Pull request merged**